resume/src/app/app.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { UserCardComponent } from './user-card/user-card.component';
+import { BadgeComponent } from './shared/badge/badge.component';
+import { LanguageCardComponent } from './language-card/language-card.component';
+import { SkillsCardComponent } from './skills-card/skills-card.component';
+import { SummaryCardComponent } from './summary-card/summary-card.component';
+import { ExperienceCardComponent } from './experience-card/experience-card.component';
+import { EducationCardComponent } from './education-card/education-card.component';
```

Comment:
I had to comment out this reference and the one in line 23, the folder isn't part of the commit